product_docs/docs/biganimal/release/reference/access_key/index.mdx

```diff
@@ -0,0 +1,41 @@
+---
+title: "Access key"
+---
+
+Access key is a new authentication process for the BigAnimal users to access the BigAnimal API directly without the `OAuth2` authorization flow. The Access key links to one user only. Each Access key link is immutable since its creation. Each Access Key has an expiration time from one to twelve months (365 days), as customized by users.
+
```

Comment:
I think this section probably worth a table like the below screenshot @zhangcz828 found from Azure api key doc:


![image](https://github.com/EnterpriseDB/docs/assets/79249661/e2086975-35ac-4b04-97c4-c3788dd6bed4)

to explicitly tell the difference between normal user's Access key and machine user's access key. 

The table could contain columns like

* user type 
* maximum count (the quota)
* how created and managed